data/users/staff.txt

```diff
@@ -1,9 +1,9 @@
-9d913c0a-3d57-4ce9-8b7d-689973312856 // peanut
+9d913c0a-3d57-4ce9-8b7d-689973312856 // Peanut
 4f881abb-cde8-41f5-b466-482451915538 // househousehouse1
-1bd9b3b3-4a27-47c6-b2b1-4da3a62c57b3 // sudofox
-669140aa-94d8-4a1d-9c1e-431b42c4c830 // magicstar
+1bd9b3b3-4a27-47c6-b2b1-4da3a62c57b3 // Sudofox
+669140aa-94d8-4a1d-9c1e-431b42c4c830 // MagicStar
 5e7fd44d-e39f-49a7-816a-390515ee1075 // 0x22
-a4f77739-d15e-4dc2-b957-219a2f6f9244 // leafhacker 
-93048fb2-d9fe-413b-9889-6dfaf04d13a5 // olibols
-7965d541-c427-4dd8-9607-e7ed5dd84b61 // pixel
-cb25290a-c504-4758-8223-27c6f9fa8c7a // slayerfox
+a4f77739-d15e-4dc2-b957-219a2f6f9244 // LeafHacker 
+93048fb2-d9fe-413b-9889-6dfaf04d13a5 // Olibols
+7965d541-c427-4dd8-9607-e7ed5dd84b61 // Pixel
+cb25290a-c504-4758-8223-27c6f9fa8c7a // Slayerfox
```

Comment:
@ZeroMemes do other files (like `pepsi.txt` support `// comments`? Or is this a new feature in 3.1 that would break 3.0?)